src/components/icons/Location.tsx

```diff
@@ -0,0 +1,25 @@
+import BaseIcon, { BaseIconProps } from './BaseIcon';
```

Comment:
This was added to Flora, so you can just use it from there: `ESystemIconNames.LOCATION`

Reply:
sorry, to clarify the icon was added to flora so you can remove this whole icon if using the Flora one fits your use case.

Reply:
good catch, yeah this was accidentally left in from a while ago